lib/Dialect/LinalgTransform/IR/PDL.cpp

```diff
@@ -293,12 +297,15 @@ static LogicalResult isDimDynamic(PDLValue value, ArrayAttr constantParams,
   return success(shapedType && shapedType.isDynamicDim(dim));
 }
 
-FailureOr<SmallVector<Operation *>> findMatchingOps(Operation *op,
+FailureOr<SmallVector<Operation *>> findMatchingOps(transform::MatchOp matchOp,
                                                     SymbolRefAttr pattern,
-                                                    ModuleOp module) {
-  auto patternOp = module.lookupSymbol<pdl::PatternOp>(pattern);
+                                                    Operation *containerOp) {
+  auto moduleOp = matchOp->getParentOfType<ModuleOp>();
+  if (!moduleOp)
+    return {moduleOp->emitError("no parent module op")};
+  auto patternOp = moduleOp.lookupSymbol<pdl::PatternOp>(pattern);
```

Comment:
Can we just find the closest symbol table to `matchOp` and look up in it? This will be truly independent of ModuleOp.